app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,91 @@
+class QuestionsController < ApplicationController
+  skip_before_action :ensure_logged_in, only: [:index, :show]
```

Comment:
It looks like the before_action in ApplicationController is missing so this doesn't do anything.